api/grpc/server_integ_test.go

```diff
@@ -310,51 +355,79 @@ func OpenPayCh(t *testing.T, sessionID string, parts, bal []string, wantErr bool
 	require.NoErrorf(t, err, "OpenPayCh")
 
 	if wantErr {
-		t.Logf("%+v", resp)
 		errMsg, ok := resp.Response.(*pb.OpenPayChResp_Error)
 		require.True(t, ok, "OpenPayCh returned success response")
 		t.Log(errMsg)
 		return ""
 	}
 	msg, ok := resp.Response.(*pb.OpenPayChResp_MsgSuccess_)
-	require.True(t, ok, "OpenPayCh returned error response")
+	assert.True(t, ok, "OpenPayCh returned error response")
 	return msg.MsgSuccess.OpenedPayChInfo.ChID
 }
 
-func SubPayChProposal(t *testing.T, sessionID string) pb.Payment_API_SubPayChProposalsClient {
+func SubPayChProposal(t *testing.T, sessionID string, wantErr bool) pb.Payment_API_SubPayChProposalsClient {
 	subReq := pb.SubPayChProposalsReq{
 		SessionID: sessionID,
 	}
 	subClient, err := client.SubPayChProposals(ctx, &subReq)
+	// if wantErr {
```

Comment:
Delete the `if` condition if not required.

Reply:
Done